app/main/views/suppliers.py

```diff
@@ -48,13 +48,13 @@ def get_supplier(code):
 
     if request_wants_json():
         return jsonify(dict(supplier))
-    if feature.is_active('SELLER_REGISTRATION'):
+    if feature.is_active('NEW_SELLER_PROFILE'):
         # add business/authorized representative contact details
         if len(supplier['contacts']) > 0:
             supplier['email'] = supplier['contacts'][0]['email']
             supplier['phone'] = supplier['contacts'][0]['phone']
             supplier['representative'] = supplier['contacts'][0]['name']
-        props = {"application": supplier}
+        props = {"application": {key: supplier[key] for key in supplier if key not in ['disclosures']}}
```

Comment:
Doesn't send disclosure info which shouldn't be displayed when viewing through this route. Otherwise even if isn't displayed, the values are in the props blob.